pgxn/neon/libpagestore.c

```diff
@@ -694,7 +694,7 @@ call_PQgetCopyData(shardno_t shard_no, char **buffer)
 		WaitEvent	event;
 		long		timeout;
 
-		timeout = Min(0, LOG_INTERVAL_US - INSTR_TIME_GET_MICROSEC(since_last_log));
+		timeout = Max(0, LOG_INTERVAL_MS - INSTR_TIME_GET_MILLISEC(since_last_log));
```

Comment:
It doesn't work, because `LOG_INTERVAL_MS` is unsigned, so if it is smaller than `INSTR_TIME_GET_MILLISEC(since_last_log))` this expression will result very large value